recipes/aravis/all/conanfile.py

```diff
@@ -87,10 +103,14 @@ def validate(self):
             )
 
     def build_requirements(self):
-        self.tool_requires("meson/1.2.1")
+        #windows build: meson/1.2.1 works, meson/1.2.2 breaks for some reason!
+        if self.settings.os == "Windows":
+            self.tool_requires("meson/[<1.2.2]")
+        else:
+            self.tool_requires("meson/1.2.2")
         self.tool_requires("glib/<host_version>")
         if not self.conf.get("tools.gnu:pkg_config", check_type=str):
-            self.tool_requires("pkgconf/1.9.5")
+            self.tool_requires("pkgconf/2.0.3")
         if self.options.introspection:
             self.tool_requires("gobject-introspection/1.72.0")
```

Comment:
The cherry-picked PR has this moved to a normal requires. I'm thinking it was an error on the original PR, but I'd like to double check first :)

Reply:
yes, on windows we need pkg-config to find glib.